src/components/tag-input/tag-input.ts

```diff
@@ -65,6 +65,7 @@ export class TagInput {
         const tagsValue = this.routeHelper.getTags();
 
         if (!tagsValue) {
+            this.onChange([]);
```

Comment:
There should be a check if `onChange` is actually defined before invoking it. There is similar check below.

Also, probably this logic here need to be revised, because calling `onChange` when the widget just being initialized doesn't seem right.

Reply:
Good point, I will make the check.

Regarding the logic:
I have tried to make a different event at first, `onLoaded()` that will notify the components only when the tag-input finished loading the list, but I noticed that it's exactly the same logic from the `onChange()` so I decided to use it, even if the name is not very suggestive for this case.

We can also make the api-list to get the tags and do the search according to them and send the tags to the tag-input, in order the component ( to be able to select tags from the UI)

Reply:
Good point, I will make the check.

Regarding the logic:
I have tried to make a different event at first, onLoaded() that will notify the components only when the tag-input finished loading the list, but I noticed that it's exactly the same logic from the onChange() so I decided to use it, even if the name is not very suggestive for this case.

We can also make the api-list to get the tags and do the search according to them and send the tags to the tag-input, in order the component ( to be able to select tags from the UI)